.clang-format

```diff
@@ -19,14 +19,17 @@ BreakBeforeTernaryOperators: 'true'
 BreakConstructorInitializersBeforeComma: 'true'
 Cpp11BracedListStyle: 'true'
 KeepEmptyLinesAtTheStartOfBlocks: 'false'
-Language: Cpp
 NamespaceIndentation: All
 PenaltyBreakString: '3'
-SpaceBeforeParens: Never
-SpacesInAngles: 'true'
+SpaceBeforeParens: Always
+SpacesInAngles: 'false'
 SpacesInContainerLiterals: 'false'
 SpacesInParentheses: 'false'
 SpacesInSquareBrackets: 'false'
 Standard: Cpp11
 UseTab: Never
-SortIncludes: false
\ No newline at end of file
+SortIncludes: false
+ColumnLimit: 100
+
+# when wrapping function calls/declarations, force each parameter to have its own line
+BinPackParameters: 'true'
```

Comment:
Example:
```c++
void foo(int arg1,
         int arg2,
         int arg3);
```

Notice that this isn't allowed with this rule:
```c++
void foo(int arg1,
         int arg2, int arg3);
```

If such a declaration/function call wraps, this rule enforces that each param is on its own line. Feel free to object.